src/utils.ts

```diff
@@ -0,0 +1,46 @@
+export const BTC = 'BTC'
+export const SATS = 'sats'
+
+export const serialize = (form: HTMLFormElement) => Object.fromEntries(new FormData(form).entries())
+
+export const walletDisplayName = (name: string) => name.replace('.jmdat', '')
+
+export const displayDate = (string: string) => new Date(string).toLocaleString()
+
+export const btcToSats = (value: string) => Math.round(parseFloat(value) * 100000000)
+
+export const satsToBtc = (value: string) => parseInt(value, 10) / 100000000
+
+export const copyToClipboard = (
```

Comment:
Should this function be async?

Reply:
I generally prefer async/await, as I find it more readable. 

But I was suggesting changing this
```suggestion
export const copyToClipboard = async (
```
because it might return a promise

Reply:
> because it might return a promise

`copyToClipboard` returns `Promise<boolean>` -> You can always use `await` in any `async` function calling it. Hope it is acceptable not using `async`/`await` in the function itself.